packages/teleport/src/Audit/__snapshots__/Audit.story.test.tsx.snap

```diff
@@ -4121,7 +4121,7 @@ exports[`list of all events 1`] = `
         <td
           style="word-break: break-word;"
         >
-          User [root] failed to query attributes of file [/tmp/file] on node [[::1]:3022]
+          User [root] failed to query attributes of file [/tmp/file] on node [[::1]:3022]: [EOF]
```

Comment:
it looks a bit weird to wrap error messages in a bracket, especially since we have a `:` separator to mean anything coming after it is a error message. i don't think there is a hard rule that every param needs to be in a bracket (it just so happens we do have pretty much everything wrapped in bracket but i think that's to make it like a visual separator from other words)

Reply:
That makes sense, I originally wrapped the error in brackets because I noticed that's what SCP events did as well. I can change that, but if I do there should probably be a follow up PR that makes formatting consistent between all failure events.

Reply:
ah ok, i missed that, lets just leave it as is then, it's not that big of a deal